src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -302,6 +296,27 @@ function parseEndpointForBuild(
     };
     if (ep.scheduleTrigger.retryConfig) {
       st.retryConfig = {};
+      convertIfPresent(
+        st.retryConfig,
+        ep.scheduleTrigger.retryConfig,
+        "maxBackoffSeconds",
+        "maxBackoffDuration",
+        (duration) => (duration === null ? null : secondsFromDuration(duration))
```

Comment:
No need to change, but you can use `nullsafeVisitor(secondsFromDuration)` if you really want. In the GCP codebase I just did

```
const toSeconds = nullsafeVisitor(secondsFromDuration)
```

and passed `toSeconds` to all converters.